### ISSUE TITLE
Bug Report: 0.7.0 - The program start by default with UI in English

### ISSUE THREAD

**Issue by bovirus** [Timthreetwelve/MyScheduledTasks#30]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened or didn't happen

@Timthreetwelve 

My OS is Windows 10 23H2 Pro 64bit italian.

After installed 0.7.0 it started with UI in English.

My expectation is that program start by default with same language of the OS (Italian).

Thanks.

### Version

0.7.0

### Windows Version

Windows 10 23 H2 Pro Italian

### Relevant log output

Install the program
Start the program.

**Comment by Timthreetwelve** [comment]
@bovirus 

![default](https://github.com/user-attachments/assets/94a7933a-250c-401d-8755-10f074b3af46)
I could change the default for the `Use Windows display language` option (in green box) to true (checked) to do this. There is already logic in place that falls back to en-US if the language translation doesn't exist.

Does that sound like it would address your issue?

**Comment by bovirus** [comment]
@Timthreetwelve 

Of course it's possible to change the language via Settings,.
My expactation is that by defaulkt the option "Use Windows display language" was set,.
Then please make this change.

**Comment by Timthreetwelve** [comment]
Yes, it will still be possible to change the language via Settings.

I will make this change to each of my multilingual apps. I'm currently working on updating HWiNFO VSB Viewer. It will be the first release to have this change.

Thanks for the suggestion.

Tim

**Comment by bovirus** [comment]
Thanks.
